agenta-backend/agenta_backend/models/db_engine.py

```diff
@@ -19,7 +17,7 @@ class DBEngine(object):
     """
 
     def __init__(self) -> None:
-        self.mode = os.environ.get("DATABASE_MODE", "v2")
+        self.mode = os.environ["DATABASE_MODE"]
```

Comment:
@aybruhm since you removed this do you wanna add this in the docker-compose ?

Reply:
Oh no, it is located in `agenta-backend/agenta_backend/config.py` at line 18.